pyramid_zipkin/request_helper.py

```diff
@@ -53,14 +37,36 @@ def get_trace_id(request):
     completely random trace id.
 
     :param: current active pyramid request
-    :returns: string representaiton of zipkin trace id
+    :returns: a 64-bit hex string
     """
     if 'X-B3-TraceId' in request.headers:
-        return request.headers['X-B3-TraceId']
+        id = request.headers['X-B3-TraceId']
```

Comment:
I'd avoid overloading the `id` variable. If someone, for whatever arcane reason, tries to access an object's location in memory later in this function it'd fail :P.